stempy/equalizer.h

```diff
@@ -0,0 +1,15 @@
+// equalizer
+#ifndef equalizer_h
+#define equalizer_h
+
+#include "image.h"
+
+#include <vector>
+
+namespace stempy {
+
+    // function that computes histogram for all the STEM images
+    std::vector<int> createSTEMHistograms (STEMImage inImage, const int numBins);
+}
+
+#endif
```

Comment:
Think we can just add the `createSTEMHIstogram` to `image.h` and `image.cpp`, rather than having `equalizer.h` and `equalizer.cpp`

Reply:
I was implementing it in the new files because I was thinking later it might also contain equalizer stuff (thus the naming). But we can definitely put all those stuff within image.h and image.cpp if that makes more sense to you.

Reply:
My thought is that the histogram creation functionality will be useful not just for the contrast equalization, so probably belongs with the other image processing functions.

Reply:
Sure, I will then integrate equalizer.h/cpp into image.h/cpp.